conf/application-json-logger.xml

```diff
@@ -5,9 +5,9 @@
         <encoder class="uk.gov.hmrc.play.logging.JsonEncoder"/>
     </appender>
 
-    <logger name="uk.gov" level="${logger.uk.gov:-WARN}"/>
+    <logger name="uk.gov" level="WARN"/>
 
-    <root level="${logger.application:-WARN}">
+    <root name="application" level="WARN">
```

Comment:
Looking in other repo's which are new, looks like the root doesn't use a name, see income-tax-software-choices-frontend

Reply:
I added it, otherwise it wouldn't have been functionally equivalent to what was there before, as I understand it.  Happy to remove if you think it was always redundant.